platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/cli/ValidateAddressBookStateCommand.java

```diff
@@ -73,7 +73,7 @@ public Integer call() throws IOException, ExecutionException, InterruptedExcepti
         BootstrapUtils.setupConstructableRegistry();
 
         final PlatformContext platformContext = new DefaultPlatformContext(
-                configuration, new NoOpMetrics(), CryptographyHolder.get(), Time.getCurrent());
+                configuration, new NoOpMetrics(), CryptographyHolder.get(), Time.getCurrent(), null);
```

Comment:
Should this be null? I think it would be ok to instantiate a file system manager in the current working directory for this.